service/src/main/java/bio/terra/workspace/service/workspace/flight/WorkspaceFlightMapKeys.java

```diff
@@ -72,6 +70,11 @@ private ControlledResourceKeys() {}
     public static final String RESOURCE_DESCRIPTION = "resourceDescription";
     public static final String PREVIOUS_RESOURCE_DESCRIPTION = "previousResourceDescription";
     public static final String PREVIOUS_ATTRIBUTES = "previousAttributes";
+    // additional keys for job filtering
+    public static final String STEWARDSHIP_TYPE = "stewardshipType";
+    public static final String RESOURCE = "resource";
```

Comment:
would it be clearer to name it to resourceType?

Reply:
There is a RESOURCE_TYPE key already. This is the RESOURCE - the whole resource object. The goal is that we can scrape Stairway to see the resources, either in the process of being created (like notebooks), or having been deleted.